app/src/main/java/com/fstyle/androidtrainning/screen/tabhome/MovieAdapter.java

```diff
@@ -0,0 +1,86 @@
+package com.fstyle.androidtrainning.screen.tabhome;
+
+import android.support.annotation.IntDef;
```

Comment:
ERROR: Unused import: android.support.annotation.IntDef. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports